### PR TITLE
DOC: Added example for get_printoption, recfromtxt and recfromcsv

### PR DESCRIPTION
**Pull request by isthatdebbiej**

### CODE REVIEW

**Comment by isthatdebbiej** [comment]
Contributed as a part of PyData NYC 2022

**Comment by WarrenWeckesser** [comment]
Thanks @isthatdebbiej.  The [NumPy docstring standard](https://numpydoc.readthedocs.io/en/v1.5.0/format.html) specifies that the *Examples* section is the *last* section of the docstring.  Also, it should have the same indentation level as the other sections.  Finally--really nitpicky, I know--the section name is *Examples*, not *Example* (even if there is, in fact, only one example!).

**Comment by isthatdebbiej** [comment]
@WarrenWeckesser Thank you! I have updated it.

**Review by rossbar — COMMENTED** [review]
Thanks @isthatdebbiej . While looking at this I realized that the keys listed in `Returns` description are actually out-of-date - we should take this opportunity to either update them, or delete them and refer to the examples section which should be tested and thus always up-to-date.

File: numpy/lib/npyio.py [hunk 1]
```diff
@@ -2488,6 +2488,10 @@ def recfromtxt(fname, **kwargs):
     By default, `dtype` is None, which means that the data-type of the output
     array will be determined from the data.
 
+    Examples
+    --------
+    np.recfromtxt('data.txt') 
+
```
[the remark this answers is not in the document]
Comment (isthatdebbiej) [reply numpy/lib/npyio.py:2494]: I also added examples for recfromtxt and recfromcsv. Should I create a new PR for that? Or update the PR title?

[the remark this answers is not in the document]
Comment (InessaPawson) [reply numpy/lib/npyio.py:2494]: @isthatdebbiej In this case, updating the PR title and description will work.

**Comment by mattip** [comment]
I think we should close this. The examples do not add that much information.

**Comment by rossbar** [comment]
> I think we should close this. The examples do not add that much information.

Thanks @mattip . I had intended to try to expand on these examples a bit but since the PR originates from `main` on the fork, pushing to the fork is disallowed. I agree it's probably easiest to close this. Thanks @isthatdebbiej , if you're still interested in contributing I would recommend opening a new PR that:
 1. Originates from a branch dedicated to the fix (i.e. `git checkout -b add-get_printoption-example`)
 2. Focuses on a single proposed change.

**Pull request closed without merging**